cognite/extractorutils/uploader/time_series.py

```diff
@@ -83,7 +84,13 @@ def default_time_series_factory(external_id: str, datapoints: DataPointList) ->
     return TimeSeries(external_id=external_id, is_string=is_string)
 
 
-class BaseTimeSeriesUploadQueue(AbstractUploadQueue):
+def _apply_cognite_timeseries(cdf_client: CogniteClient, timeseries_apply: CogniteExtractorTimeSeriesApply) -> NodeId:
+    instance_result = cdf_client.data_modeling.instances.apply(timeseries_apply)
```

Comment:
The legacy equivalent does not create the time series in CDF itself, it only returns python objects that is then created in CDF by the queue.

I think we should try to keep the same behavior here